config.go

```diff
@@ -29,45 +32,160 @@ const (
 
 	uiPasswordMinLength = 8
 
-	defaultLndMacaroon = "admin.macaroon"
+	ModeIntegrated = "integrated"
+	ModeRemote     = "remote"
+
+	defaultLndMode = ModeRemote
```

Comment:
should this be integrated?

Reply:
No, we decided that the default mode should be remote as most users might already be running an `lnd` node.

Reply:
Ok, then I think maybe the config description must be updated, as it says `integrated` is the default!